Frontend/ui-customer/src/Helpers/Utils.purs

```diff
@@ -387,8 +387,5 @@ makeNumber number = (DS.take 2 number) <> " " <> (DS.drop 2 (DS.take 4 number))
 reverse' :: String -> String
 reverse' = fromCharArray <<< reverse <<< toCharArray
 
-getMerchantVechicleSize :: Unit -> Int
-getMerchantVechicleSize unit = 
- case getMerchant FunctionCall of
-   YATRI -> 160
-   _ -> 90
+getVehicleSize :: Unit -> Int
+getVehicleSize unit = 90
```

Comment:
@prakharritik please replace the function name wherever it is being used.

Reply:
Already done.